.gitignore

```diff
@@ -4,3 +4,4 @@
 *.swn
 .vscode
 node_modules/
+*/prod-relayer-config.json
```

Comment:
does this work to exlcude a sub dir? bc i think it will be libs/initScripts/prod-relayer-config.json and I thought you ned like */**/prod-relayer-config.json or something?

Reply:
reverted this change - i actually added to the wrong .gitignore, the libs one already ignores this file